packages/server-core/src/projects/project/project.class.ts

```diff
@@ -220,8 +221,11 @@ export class Project extends Service {
     })
     if (existingProjectResult != null) await super.remove(existingProjectResult.id, params)
 
+    const repoPath = await getAuthenticatedRepo(data.url)
```

Comment:
we should still allow public repos to be cloned if unauthenticated. lets let this fail gracefully and notify the user they are unauthorised if the repo is private.